src/components/TopBarMenu/Cart/Cart.jsx

```diff
@@ -1,17 +1,51 @@
-import { useContext } from "react";
+import { useState, useContext } from "react";
 import CartContext from "./CartContext.js";
+import Images from "../../Images/Images.jsx";
+import Button from "../../Button/Button.jsx";
 
 import "./Cart.scss";
 
 export default function Cart() {
-  const { cart } = useContext(CartContext);
+  const { cart, setCart } = useContext(CartContext);
+  const [cartBox, setCartBox] = useState("cartboxOff");
+
+  function handleChangeCartbox() {
+    setCartBox(cartBox === "cartboxOff" ? "cartboxOn" : "cartboxOff");
+  }
+
+  function removeItemCart(cartQuantity) {
+    setCart({ product: null, quantity: 0, value: "false" });
+  }
 
   return (
-    <div className="teste">
-      <img src={`/images/${cart.product?.thumbnail}`} alt="" />
-      <p>{cart.product?.model}</p>
-      <p>{cart.quantity}</p>
-      <p>{cart.product?.price}</p>
-    </div>
+    <>
+      <Images
+        onClick={handleChangeCartbox}
+        source="icon-cart.svg"
+        imagetype="cart"
+        alt="cart icon"
+      />
+      <div id={cartBox}>

```

Comment:
Always prefer to use `class` instead of `id`.

What you can do here it's a nice thing to use data-attribute:

```jsx
<div class='cartBox' data-status={cartBoxStatus}>
```

and inside your CSS:
```scss
.cartbox {
  flex-direction: column;
  position: absolute;
  left: 0;
  background-color: white;
  border-radius: 10px;
  box-shadow: 0 0 1em black;
  height: fit-content;
  padding: 10px;
  z-index: 999;
  min-width: 250px;
  
  [data-status="on"] {
    display: flex; // you can move it to the `.cartbox` class.
  }

  [data-status="off"] {
    display: none;
  }
}
```